src/style/themes/sage/index.ts

```diff
@@ -0,0 +1,18 @@
+import tokens from "@sage/design-tokens/js/base/common";
+import { mergeWithBase } from "../base";
+import { BasePalette } from "../base/base-theme.config";
+
+export default {
+  ...mergeWithBase((palette: BasePalette) => ({
+    colors: {
+      primary: palette.productGreenShade(23.5),
+      secondary: palette.productGreenShade(41),
+      tertiary: palette.productGreenShade(61),
+      loadingBarBackground: palette.productGreenTint(70),
+    },
+  })),
+  compatibility: {
+    ...tokens,
+  },
```

Comment:
**comment**: I noticed a token had a different value after this change.

It seems previously we overrode certain tokens within `compatibility` within the `base` theme:

```ts
{
  name: "base",
  get compatibility() {
      return {
        ...tokens,

        /* Base theme overrode these tokens: */
        colorsActionMajor150: this.colors.loadingBarBackground,
        colorsActionMajor500: this.colors.primary,
        colorsActionMajor600: this.colors.secondary,
        colorsActionDisabled500: this.disabled.background,
        colorsSemanticFocus500: this.colors.focus,
        colorsSemanticNegative500: this.colors.error,
        colorsSemanticNegative600: this?.colors?.destructive?.hover,
        colorsSemanticCaution500: this.colors.warning,
        colorsSemanticInfo500: this.colors.info,
      };
    },
}
```

but we are now setting these tokens to their original consumed values from `@sage/design-tokens`:

```ts
{
  name: "sage",
  ...mergeWithBase((palette: BasePalette) => ({
      /* deep merge with base */
  })),
  compatibility: { /* 👈 overrides `compatibility` of `base` completely */
    ...tokens,
  },
}
```

The token value changes don't seem significant, so happy to approve this to eliminate complexity. But wanted to highlight this in case QA had any queries about certain chromatic changes.